### PR TITLE
Upgrader

### PR DESCRIPTION
**Pull request by semseysandor**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: CI
 
 on:
   pull_request:
-    branches: [ main ]
+#    branches: [ main ]
```
Comment (akosgarai) [inline .github/workflows/main.yml:5]: Please apply this line again.

Comment (semseysandor) [reply .github/workflows/main.yml]: done

File: api_subscription.php [hunk 2]
```diff
@@ -31,10 +31,13 @@ function api_subscription_civicrm_xmlMenu(&$files)
  * Implements hook_civicrm_install().
  *
  * @link https://docs.civicrm.org/dev/en/latest/hooks/hook_civicrm_install
+ * @throws \CRM_Core_Exception
  */
 function api_subscription_civicrm_install()
 {
     _api_subscription_civix_civicrm_install();
+    $installer = _api_subscription_civix_upgrader();
```
Comment (akosgarai) [inline api_subscription.php:39]: Why do you need this installer and call the install function? As far as i see, it is already handled in the `_api_subscription_civix_civicrm_install` function.

File: api_subscription.php [hunk 3]
```diff
@@ -51,10 +54,13 @@ function api_subscription_civicrm_postInstall()
  * Implements hook_civicrm_uninstall().
  *
  * @link https://docs.civicrm.org/dev/en/latest/hooks/hook_civicrm_uninstall
+ * @throws \CRM_Core_Exception
  */
 function api_subscription_civicrm_uninstall()
 {
     _api_subscription_civix_civicrm_uninstall();
+    $installer = _api_subscription_civix_upgrader();
```
Comment (akosgarai) [inline api_subscription.php:62]: Why do you need this installer and call the uninstall function? As far as i see, it is already handled in the `_api_subscription_civix_civicrm_uninstall` function.

File: api_subscription.php (hunk 2 above)
Comment (semseysandor) [reply api_subscription.php:39]: That's true, I haven't looked into that very deeply.

File: api_subscription.php (hunk 3 above)
Comment (semseysandor) [reply api_subscription.php]: same as above

**Review by akosgarai — APPROVED** [review]

**Pull request merged**